components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.html

```diff
@@ -64,37 +64,32 @@ <h3>Check-in History <span *ngIf="showCheckinDebug">(debug)</span></h3>
         </chef-tr>
       </chef-thead>
       <chef-tbody>
-        <chef-tr *ngFor="let item of checkinHistory; index as i">
-          <chef-td class="status-cell" [id]="'checkin-item-'+i+'text'">
-            <chef-icon>{{ item.icon }}</chef-icon>
-            <span>{{ item.text | titlecase }}</span>
+        <chef-tr *ngFor="let history of checkInHistory; index as i">
+          <chef-td class="status-cell">
+            <chef-icon [id]="'checkin-item-'+i+'text'" [ngClass]="history.status">
+              {{ historyIcons[history.status] }}
+            </chef-icon>
+            <span>{{ history.status | titlecase }}</span>
             <chef-tooltip
-              [ngClass]="['checkin-tooltip', item.text]"
-              [attr.for]="'checkin-item-'+i+'text'">
+              [ngClass]="['checkin-tooltip', history.status]"
+              [attr.for]="'checkin-item-'+i+'text'"
+              interactable>
               <p class="checkin-tooltip-info">
-                <span class="checkin-tooltip-text">{{ item.text | titlecase }}</span>
-                <span class="checkin-tooltip-date">{{ item.date | datetime: DateTime.RFC2822 }}</span>
+                <span class="checkin-tooltip-text">{{ history.status | titlecase }}</span>
+                <span class="checkin-tooltip-date">{{ history.end | datetime: DateTime.RFC2822 }}</span>
               </p>
-              <a class="checkin-tooltip-link" href="">View Client Run Details <chef-icon>open_in_new</chef-icon></a>
+              <a class="checkin-tooltip-link" [routerLink]="['/infrastructure/client-runs', desktop.id, 'runs', history.run_id]">View Client Run Details <chef-icon>open_in_new</chef-icon></a>
```

Comment:
? for ux peeps: since we're using an 'open in new' icon, should we be opening this link in a new browser tab?

Reply:
It seems `history.run_id` is `undefined` for history items with a `missing` status. I suppose we should we remove the 'view detail' link on `missing` items?

Reply:
@scottopherson , correct. If the status for a day is "unknown", there shouldn't be a link, instead just a label of "unknown".
![image](https://user-images.githubusercontent.com/9401459/83185872-843e1100-a0e0-11ea-9eb5-b3543f4d74b4.png)

Reply:
For those that have an external link to the client run details, it should open up in a new browser tab.

Reply:
Demo for the intended behavior: https://drive.google.com/file/d/1nfGs5SH9RUHGBGdjBtYB96LtKRlMY135/view

Reply:
@apriofrost Do we need to open in new window?